internal/util/ip_utils_test.go

```diff
@@ -121,3 +123,43 @@ func TestAppendPrefixMask(t *testing.T) {
 		})
 	}
 }
+
+// TestIsIPv4DefaultRoute tests the IsIPv4DefaultRoute function.
+func TestIsDefaultIPv4Route(t *testing.T) {
+	testCases := []struct {
+		input    string
+		expected bool
+	}{
+		{"0.0.0.0", true},
+		{"0.0.0.0/0", true},
+		{"192.168.1.1", false},
+		{"192.168.1.1/24", false},
```

Comment:
I'd add a test case with garbage input. Right now this code is called on invalidated user input. I'd like to change that so the input is validated before it gets this far, but might as well make sure the code doesn't blow up (now or after future changes) on garbage.